css/home-page-for-user.styl

```diff
@@ -55,6 +55,31 @@ GRAY = #EFF2F5
     padding: 1em 2em
     text-align: left
 
+  &__change-background
+    border: 1px solid white
+    border-radius: 0.8em
+    cursor: pointer
+    font-weight: bold
+    padding: 0.8em
+    position: relative
```

Comment:
This should be `position: absolute` and `top` and `left` instead. Right now, the button will move around depending on the screen size and can potentially overlap with other elements in the nav bar. Setting the margin is unnecessary with absolute positioning.